src/promptflow/promptflow/_sdk/schemas/_experiment.py

```diff
@@ -30,6 +36,11 @@ class Meta:
     inputs = fields.Dict(keys=fields.Str)
     path = UnionField([LocalPathField(required=True), fields.Str(required=True)])
 
+    @pre_load
+    def warning_unknown_fields(self, data, **kwargs):
```

Comment:
What unknown fileds for node that does not need warning? Besides, this implementation is kinda werid, we should set `unknown="exclude"` at the place where the schema being used, or you change the original `warning_unknown_fields` function to skip for non-runschema instances?